models/twofactor.go

```diff
@@ -61,7 +61,7 @@ func (t *TwoFactor) getEncryptionKey() []byte {
 
 // SetSecret sets the 2FA secret.
 func (t *TwoFactor) SetSecret(secret string) error {
-	secretBytes, err := com.AESEncrypt(t.getEncryptionKey(), []byte(secret))
```

Comment:
The changes to 2FA are unrelated to this PR, could they be pulled out an resubmitted under a diff PR?

Reply:
They are necessary, because of the macaron update.

Reply:
Thanks for the clarification :)